pkg/controller/volume/expand/expand_controller.go

```diff
@@ -370,6 +371,10 @@ func (expc *expandController) isNodeExpandComplete(logger klog.Logger, pvc *v1.P
 }
 
 // Implementing VolumeHost interface
+func (expc *expandController) GetTracerProvider() trace.TracerProvider {
+	return nil
```

Comment:
Why is there no TracerProvider here?

Reply:
The `expand` package is used by `kube-controller-manager`, and since this pr is focusing on add trace for csi-plugin in `kubelet`, so I do not implement the `GetTracerProvider` interface used by `kube-controller-manager.`

Reply:
Please use a Noop instead of nil

Reply:
Done